core-plugin/src/com/google/gct/idea/debugger/ui/ExitDialog.java

```diff
@@ -0,0 +1,44 @@
+package com.google.gct.idea.debugger.ui;
+
+import com.google.gct.idea.debugger.CloudDebugProcessWatcher;
+import com.google.gct.idea.util.GctBundle;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.DialogWrapper;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+
+/**
+ * Dialog shown when user presses the red stop button to
+ * ask whether IntelliJ should continue to listen for snapshots.
+ */
+public class ExitDialog extends DialogWrapper {
+
+  private final JLabel myLabel = new JLabel(GctBundle.getString("clouddebug.continuelistening"));
+
+  public ExitDialog(@Nullable Project project) {
+    super(project, false, IdeModalityType.MODELESS);
+
+    init();
+
+    setOKButtonText(GctBundle.getString("clouddebug.continue"));
+    setCancelButtonText(GctBundle.getString("clouddebug.stoplistening"));
+  }
```

Comment:
can you add the tracking to differentiate when a user opts to stop vs continue listening?